start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
+
+# Problem
+
+An InnerSource initiative is considered but not started because management is 
+unsure about its outcome and, as a result, is not willing to commit to an 
+investment.
+
+# Context
+
+The majority of software development projects in the company are distributed
+globally. The company is considering to leverage InnerSource to make the
+required collaboration more efficient. However, most managers are not familiar
+with the Open Source working model and are instead accustomed to hierarchical,
+top-down control style management. The idea of InnerSource is very popular with
+software developers in the company, not the least because many developers use
+or are actively developing Open Source software.
+
+# Forces
+
+- Managers will want to validate the claims of improved collaboration through
+  InnerSource before making a long term investment. This usually involves
+  putting numbers against the expected improvements, which are hard to come
+  by for InnerSource.
+- If the InnerSource initiative has a huge uptake among developers and if many
+  projects come to rely on it, a decision to shut it down will be very 
+  unpopular and therefore hard to make.
```

Comment:
> If many projects come to rely on InnerSource, doesn't that mean that it is successful?

That would be my definition of success, yes. But then, I'm not a manager. The benefits of InnerSource can sometimes be hard to quantify, especially the soft factors, such as employee satisfaction, improved learning and increased networking. Not being able to put hard numbers on e. g. the increase in efficiency might make InnerSource look like a failure to some managers.

> Would a decision to shut it down then be very unlikely?

I think there are shades of grey, here. If the company does not support InnerSource on the highest level anymore (e. g. no funding for central platform), the projects which are relying on InnerSource would most likely maintain a more islolated version of it for their particular use. I guess you could call that InnerSource, too, but it would probably be a step back from the values of _openness_ and _transparency_. 

> Does this force represent a fear that management has about even starting this (a worry that they would lose control over the program because of its popularity)?

Yes!